### PR TITLE
we need to add these two in order to get most of the queries right.

### PR DESCRIPTION
**Pull request by StigNorland**

### CODE REVIEW

File: search-commons/src/main/java/no/unit/nva/search/resource/ResourceParameter.java [hunk 1]
```diff
@@ -426,31 +432,4 @@ public String errorMessage() {
...
-
-    private static boolean ignoreInvalidKey(ResourceParameter enumParameter) {
-        return enumParameter.ordinal() > IGNORE_PARAMETER_INDEX;
-    }
-
-    private static boolean isSearchField(ResourceParameter enumParameter) {
-        return enumParameter.ordinal() > IGNORE_PARAMETER_INDEX
-                && enumParameter.ordinal() < AGGREGATION.ordinal();
-    }
 }
```
Comment (StigNorland) [inline search-commons/src/main/java/no/unit/nva/search/resource/ResourceParameter.java:435]: Gjort av google-autoformatterer

File: search-commons/src/main/java/no/unit/nva/search/resource/ResourceParameter.java [hunk 2]
```diff
@@ -266,7 +240,7 @@ public enum ResourceParameter implements ParameterKey<ResourceParameter> {
     TAGS_NOT(TEXT, NOT_ALL_OF, ENTITY_TAGS),
     TAGS_SHOULD(TEXT, ANY_OF, ENTITY_TAGS),
     TAGS_EXISTS(EXISTS, ANY_OF, ENTITY_TAGS),
-    TITLE(TEXT, ENTITY_DESCRIPTION_MAIN_TITLE, PI),
+    TITLE(TEXT, ALL_OF, ENTITY_DESCRIPTION_MAIN_TITLE, PI),
```
Comment (LarsV123) [inline search-commons/src/main/java/no/unit/nva/search/resource/ResourceParameter.java:243]: Is `ALL_OF` intended here?

**Review by LarsV123 — APPROVED** [review]
We should probably add the new parameters to the OpenAPI spec, but that's out of sync anyway so we could do it in a separate PR.

File: search-commons/src/main/java/no/unit/nva/search/resource/ResourceParameter.java (hunk 2 above)
Comment (StigNorland) [reply search-commons/src/main/java/no/unit/nva/search/resource/ResourceParameter.java:243]: I removed the constructor, as it made no sense to have it, besides, I think it is better to declare `AND & OR` queries explicit.

**Pull request merged**